scripts/py_matter_idl/matter_idl/data_model_xml/handlers/__init__.py

```diff
@@ -29,6 +39,11 @@ def __init__(self, context: Context, idl: Idl):
 
     def GetNextProcessor(self, name, attrs):
         if name.lower() == 'cluster':
-            return ClusterHandler(self.context, self._idl, attrs)
+            if contains_valid_cluster_id(attrs):
+                return ClusterHandler.ForAttributes(self.context, self._idl, attrs)
+
+            LOGGER.info("Found a base cluster (no id): '%s'", attrs['name'])
```

Comment:
A cluster can be a base cluster, but also usable as-is (e.g. operational state cluster is used directly in some device types, but is derived for some other uses). I think we should make sure to separate the cases of "cluster used as base sometimes"  versus "this is an abstract base class-like cluster (e.g. Mode Base, Concentration Measurement)".

Reply:
I have that logic, will test it as part as validating the bridge information cluster (I think it derives from basic information cluster).

Reply:
Also slightly updating naming to say 'abstract' to make it clear that these are abstract but other clusters could be base as well.